sormas-api/src/main/resources/captions.properties

```diff
@@ -324,6 +324,7 @@ CaseData.clinicalConfirmation=Clinical confirmation
 CaseData.community=Responsible community
 CaseData.epidemiologicalConfirmation=Epidemiological confirmation
 CaseData.laboratoryDiagnosticConfirmation=Laboratory diagnostic confirmation
+CaseData.confirmedCaseClassification = Confirmed case classification
```

Comment:
Call this property `caseConfirmationBasis` and use the caption "Basis for confirmation"